metrics.go

```diff
@@ -115,6 +115,26 @@ func protoToMetricPoint(value interface{}) (*monitoringpb.TypedValue, error) {
 			}
 		}
 		tval = &monitoringpb.TypedValue{Value: mv}
+
+	case *metricspb.Point_SummaryValue:
+		var sum float64 = v.SummaryValue.GetSum().GetValue()
+		if descriptorType == metricspb.MetricDescriptor_CUMULATIVE_DOUBLE {
```

Comment:
If value is of type `Summary`, descriptor can only be of type [`Summary`](https://github.com/census-instrumentation/opencensus-proto/blob/master/src/opencensus/proto/metrics/v1/metrics.proto#L113) too. Also see [Java](https://github.com/census-instrumentation/opencensus-java/blob/master/api/src/main/java/io/opencensus/metrics/export/Metric.java#L130-L132).

Reply:
@songy23 I read the comment in https://github.com/census-instrumentation/opencensus-proto/blob/b11a67434194733b34c11f206938263fa16ad1cf/gen-go/metrics/v1/metrics.pb.go#L59-L63 and don't understand/see how Summary can be converted into Stackdriver types which is why I used *Int64 and *Double. How does Java handle this conversion in the Stackdriver exporter?

Reply:
`Summary` value has three fields: sum, count and snapshot. Snapshot has values at percentiles. IIUC in Java we convert Summary.sum and Summary.count to two TimeSeries, then convert each of the values at percentiles as a TimeSeries. I think this is what Stackdriver wants. cc @mayurkale22 for confirmation.